catroid/src/main/java/org/catrobat/catroid/io/StorageOperations.java

```diff
@@ -208,7 +208,7 @@ public static File copyDir(File srcDir, File dstDir) throws IOException {
 			throw new IOException(srcDir.getAbsolutePath() + " is not a directory.");
 		}
 
-		dstDir.mkdir();
```

Comment:
any reason for this change?

Reply:
Yes. mkdir throws an exception if the path of the Folder doesn't exist.
mkdirs creates the path and the folder, so you don't have to look if the CATROBAT Folder exists or not. It makes the code simple and doesn't change the logic that much.